helm/aws-operator/values.yaml

```diff
@@ -19,8 +19,8 @@ pod:
   group:
     id: 1000
 project:
-  name: "aws-operator"
-  version: "[[ .Version ]]"
+  branch: "[[ .Branch ]]"
+  commit: "[[ .SHA ]]"
 # Resource names are truncated to 47 characters. Kubernetes allows 63 characters
 # limit for resource names. When pods for deployments are created they have
 # additional 16 characters suffix, e.g. "-957c9d6ff-pkzgw" and we want to have
```

Comment:
It would be cool to also move the stuff from `resource:` to helper. But I guess a separate PR would make sense.

Reply:
Yeah I agree, but let's allow folks to get used to the helper and do that separately. Also I'd like to close this soon given how many things it's blocking.